Header header
float32 temperature # temperature in Celsius
